File: SAMPLE/CQL/JS/blog_type.js
class SECTION
{
    constructor(
        )
    {
        this.Uuid = null;
        this.Number = null;
        this.Name = null;
        this.Text = null;
        this.Image = null;
        this.ImageIndex = null;
    }
}

// ~~

class USER
{
    constructor(
        )
    {
        this.Uuid = null;
        this.FirstName = null;
        this.LastName = null;
        this.Email = null;
        this.Pseudonym = null;
        this.Password = null;
        this.Phone = null;
        this.Street = null;
        this.City = null;
        this.Code = null;
        this.Region = null;
        this.Country = null;
        this.Company = null;
        this.ItIsAdministrator = null;
    }
}

// ~~

class ARTICLE
{
    constructor(
        )
    {
        this.Uuid = null;
        this.SectionUuid = null;
        this.UserUuid = null;
        this.Title = null;
        this.Text = null;
        this.Image = null;
        this.Date = null;
        this.Section = null;
        this.User = null;
        this.ImageIndex = null;
    }
}

// ~~

class COMMENT
{
    constructor(
        )
    {
        this.Uuid = null;
        this.ArticleUuid = null;
        this.UserUuid = null;
        this.Text = null;
        this.DateTime = null;
        this.Article = null;
        this.User = null;
    }
}

// ~~

class SUBSCRIBER
{
    constructor(
        )
    {
        this.Uuid = null;
        this.Name = null;
        this.Email = null;
    }
}

File: SAMPLE/CQL/CS/blog_type.cs
public class SECTION
{
    public Guid
        Uuid;
    public ulong
        Number;
    public string
        Name,
        Text,
        Image;
    public ulong
        ImageIndex;
}

// ~~

public class USER
{
    public Guid
        Uuid;
    public string
        FirstName,
        LastName,
        Email,
        Pseudonym,
        Password,
        Phone,
        Street,
        City,
        Code,
        Region,
        Country,
        Company;
    public bool
        ItIsAdministrator;
}

// ~~

public class ARTICLE
{
    public Guid
        Uuid,
        SectionUuid,
        UserUuid;
    public string
        Title,
        Text,
        Image;
    public DateTime
        Date;
    public SECTION
        Section;
    public USER
        User;
    public ulong
        ImageIndex;
}

// ~~

public class COMMENT
{
    public Guid
        Uuid,
        ArticleUuid,
        UserUuid;
    public string
        Text;
    public DateTime
        DateTime;
    public ARTICLE
        Article;
    public USER
        User;
}

// ~~

public class SUBSCRIBER
{
    public Guid
        Uuid;
    public string
        Name,
        Email;
}
